src/Windowing/Silk.NET.Windowing.Common/Internals/WindowImplementationBase.cs

```diff
@@ -178,6 +179,21 @@ public WindowBorder WindowBorder
 
         // Other property implementations
         public bool TransparentFramebuffer => ExtendedOptionsCache.TransparentFramebuffer;
-        public Rectangle<int> BorderSize => IsInitialized ? CoreBorderSize : default;
+
+        public bool TopMost
+        {
+            get => IsInitialized ? ExtendedOptionsCache.TopMost = CoreTopMost : ExtendedOptionsCache.TopMost;
+            set
+            {
+                if (IsInitialized)
+                {
+                    CoreTopMost = value;
+                }
+
+                ExtendedOptionsCache.TopMost = value;
+            }
+        }
+
+        public Rectangle<int> BorderSize             => IsInitialized ? CoreBorderSize : default;
```

Comment:
Just one more here :)
```suggestion
        public Rectangle<int> BorderSize => IsInitialized ? CoreBorderSize : default;
```

Reply:
Done, dunno why my IDE decided that putting that many spaces was a good idea xD